Tests in CloudLibrariesInPluginXmlTest will fail in the future when new artifact versions are released

Builds are currently failing. Looks like Endpoints made a new release. We need to fix this test to be future-proof.

```
Tests run: 6, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.006 sec <<< FAILURE! - in com.google.cloud.tools.eclipse.appengine.libraries.model.CloudLibrariesInPluginXmlTest
testObjectifyLibraryConfig(com.google.cloud.tools.eclipse.appengine.libraries.model.CloudLibrariesInPluginXmlTest)  Time elapsed: 0 sec
testServletApiLibraryConfig(com.google.cloud.tools.eclipse.appengine.libraries.model.CloudLibrariesInPluginXmlTest)  Time elapsed: 0 sec
testLibrarySize(com.google.cloud.tools.eclipse.appengine.libraries.model.CloudLibrariesInPluginXmlTest)  Time elapsed: 0.001 sec
testAppEngineApiLibraryConfig(com.google.cloud.tools.eclipse.appengine.libraries.model.CloudLibrariesInPluginXmlTest)  Time elapsed: 0.001 sec
testEndpointsLibraryConfig(com.google.cloud.tools.eclipse.appengine.libraries.model.CloudLibrariesInPluginXmlTest)  Time elapsed: 0.002 sec  <<< FAILURE!
java.lang.AssertionError: 
Expected: is "2.0.9"
     but: was "2.0.10"
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:956)
	at org.junit.Assert.assertThat(Assert.java:923)
	at com.google.cloud.tools.eclipse.appengine.libraries.model.CloudLibrariesInPluginXmlTest.testEndpointsLibraryConfig(CloudLibrariesInPluginXmlTest.java:117)
```

--- a/plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/src/com/google/cloud/tools/eclipse/appengine/libraries/model/CloudLibrariesInPluginXmlTest.java
+++ b/plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/src/com/google/cloud/tools/eclipse/appengine/libraries/model/CloudLibrariesInPluginXmlTest.java
@@ -78,7 +78,11 @@ public class CloudLibrariesInPluginXmlTest {
     assertThat(mavenCoordinates.getRepository(), is("central"));
     assertThat(mavenCoordinates.getGroupId(), is("com.google.appengine"));
     assertThat(mavenCoordinates.getArtifactId(), is("appengine-api-1.0-sdk"));
-    assertThat(mavenCoordinates.getVersion(), is("1.9.60"));
+    
+    DefaultArtifactVersion actual = new DefaultArtifactVersion(mavenCoordinates.getVersion());
+    DefaultArtifactVersion expected = new DefaultArtifactVersion("1.9.60");
+    assertTrue(actual.compareTo(expected) >= 0);
+    
     assertThat(mavenCoordinates.getType(), is("jar"));
     assertNull(mavenCoordinates.getClassifier());
 
@@ -116,7 +120,11 @@ public class CloudLibrariesInPluginXmlTest {
     assertThat(mavenCoordinates.getRepository(), is("central"));
     assertThat(mavenCoordinates.getGroupId(), is("com.google.endpoints"));
     assertThat(mavenCoordinates.getArtifactId(), is("endpoints-framework"));
-    assertThat(mavenCoordinates.getVersion(), is("2.0.11"));
+    
+    DefaultArtifactVersion actual = new DefaultArtifactVersion(mavenCoordinates.getVersion());
+    DefaultArtifactVersion expected = new DefaultArtifactVersion("2.0.11");
+    assertTrue(actual.compareTo(expected) >= 0);
+ 
     assertThat(mavenCoordinates.getType(), is("jar"));
     assertNull(mavenCoordinates.getClassifier());
     assertThat(libraryFile.getJavadocUri(),

